types/karma/karma-tests.ts

```diff
@@ -121,3 +121,29 @@ console.log(karma.VERSION);
 karma.config.parseConfig('karma.conf.js', {
     singleRun: true
 });
+
+// angular-cli sample config: https://git.io/Jvm9i
+module.exports = (config: karma.Config) => {
```

Comment:
can we please narrow this test config down to what we're expecting to test? rather than having a copy from another project.
most things should have been tested in the earlier config, so here you should only need a minimum config object with `restartOnFileChange` set

Reply:
sure, tbd, thanks!

Reply:
Done, commit amended. Thanks!